corehq/motech/dhis2/templates/dhis2/partials/dhis2_dataset_map_templates.html

```diff
@@ -85,7 +85,7 @@
 
         <div class="form-group">
             <label class="control-label col-sm-3 col-md-2 requiredField">
-                {% trans "UCR ID" %}<span class="asteriskField">*</span>
+                {% trans "UCR Report ID" %}<span class="asteriskField">*</span>
```

Comment:
user-configurable report report id?

Reply:
Yeah, I feel you @orangejenny . But if we changed it to "UCR ReportConfig ID" would that make it simpler or more confusing for REC users? I'm not sure.

(I should mention that I am deeply ashamed of this UI. It was never meant for non-Dimagi users. And after seeing how long this page gets in practice, it's awful, if not unusable. Redoing this is on my to-do list. This field should be a select dropdown where users choose a UCR by name. And we have requests that the "DataValue maps" should be a textarea for JSON, like we use for UCRs and other integration configs, so that content can be copy-pasted from a text editor.)

Reply:
Eh, I think of it as "User-Configurable-Reports [the feature] Report-ID"

> It was never meant for non-Dimagi users

I feel you.  There's definitely a place for making dumb, unpolished UIs for advanced internal users to do some obscure technical thing once or twice.  Unfortunately, sometimes those rare, obscure technical things turn out to be useful, and then since we _already_ have something that works...